helper.go

```diff
@@ -0,0 +1,190 @@
+package main
```

Comment:
```suggestion
package test_helper
```
This file should not be part of the main application since it is only used in the tests.

Reply:
I did that but in this case some other updates should take place mostly in go.mod and go.sum and replace a module in a couple of files. For that reason I think that is better to make a separate pull request for that (after that one is merged) in order to take a better look in updates. 
If you think that is too much I can just add it in this pull request.

Reply:
The quick solution then would be to merge the `helper.go` into `userauth_test.go` and remove the `helper_test.go` file.
that way it is not part of the main application.